styles.css

```diff
@@ -40,8 +40,10 @@ li {
 }
 
 .mobile-version-nav.active {
-  opacity: 1;
+  opacity: 0.8;
   pointer-events: auto;
+  filter: blur(8px);
+  -webkit-filter: blur(8px);
 }
 
```

Comment:
- [ ] Kindly make your menu button match the Figma design button. You can remove the border and the background color, and it should work.

Figma design
![image](https://user-images.githubusercontent.com/35535130/168699135-40e680b2-ecb1-4cb5-9e7d-2858ebc97650.png)

Your design
![image](https://user-images.githubusercontent.com/35535130/168699193-5481ea6b-3666-4be9-bfeb-f3f12c560635.png)